code/go/0chain.net/smartcontract/dbs/event/snapshot.go

```diff
@@ -0,0 +1,18 @@
+package event
+
+// swagger:model Block
+type Snapshot struct {
+	Round           int64  `gorm:"primaryKey;autoIncrement:false" json:"round"`
```

Comment:
it's not a good idea to make it autoIncremented, in theory we can skip rounds, sharders do not have control over round numbering

Reply:
This is autoIncrement:false. By default it's auto increment and I set it to false.